azurelinuxagent/common/cgroups.py

```diff
@@ -517,21 +516,26 @@ def setup(suppress_process_add=False):
                         cg.add(pid)
                         logger.info("Add daemon process pid {0} to {1} cgroup".format(pid, cg.name))
                     else:
-                        logger.info("Daemon process pid {0} cgroup managed by systemd".format(pid))
-                status = "OK"
+                        logger.info("Add daemon process pid {0} to systemd cgroup".format(pid))
+                status = "ok"
```

Comment:
Is the case use anywhere that it is important?

Reply:
DCR